apiary.apib

```diff
@@ -1017,6 +1017,50 @@ API endpoints that Operation Code's Rails backend makes available to its React f
             {
                 errors: "Some error message"
             }
+## Sessions | Create [/api/v1/sessions/{?email,password,first_name,last_name}]
++ Parameters
+
+      + email (string, required) - The users email
+      + password (string, required) - The users password
+      + first_name(string, required) - The users first names
+      + last_name(string, required) - The users last name
+
+### Create a Session [POST]
++ Request (application/json)
+
+    + Body
+
+            {
```

Comment:
Based on these results from the params check you did in the controller:

```
<ActionController::Parameters {"user"=>{"email"=>"mekhi_kemmer@dooleycollier.info", "password"=>"pass123"}, "controller"=>"api/v1/sessions", "action"=>"create", "session"
=>{"user"=>{"email"=>"mekhi_kemmer@dooleycollier.info", "password"=>"pass123"}}, "format"=>"json"} permitted: false>
```

The request contains:

```ruby
{"user"=>{"email"=>"mekhi_kemmer@dooleycollier.info", "password"=>"pass123"}}
```

So you'll want to update the `Body` and parameters sections accordingly.

Reply:
fixed it to user, email and password